Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -341,12 +353,56 @@ extension XPCServer {
         try XPCServiceServer._forThisXPCService()
     }
     
-    /// Creates a new anonymous server that only accepts connections from the same process it's running in.
-    internal static func makeAnonymous() -> XPCServer & NonBlockingServer {
+    /// Creates a new anonymous server that accepts requests from the same process it's running in.
+    ///
+    /// Only a client created from an anonymous server's endpoint can communicate with that server. Retrieve the ``NonBlockingServer/endpoint`` and
+    /// then create a client with it:
+    /// ```swift
+    /// let server = XPCServer.makeAnonymous()
+    /// let client = XPCClient.fromEndpoint(server.endpoint)
+    /// ```
+    ///
+    /// > Important: No requests will be processed until ``NonBlockingServer/start()`` or ``startAndBlock()`` is called.
+    ///
+    /// ## Usage
+    /// If you need this server to be communicated with by clients running in a different process, use ``makeAnonymous(clientRequirements:)`` instead.
+    public static func makeAnonymous() -> XPCServer & NonBlockingServer {
         XPCAnonymousServer(messageAcceptor: SameProcessMessageAcceptor())
     }
 
-    internal static func makeAnonymous(clientRequirements: [SecRequirement]) -> XPCServer & NonBlockingServer {
+    /// Creates a new anonymous server that accepts requests from clients which meet the security requirements.
+    ///
+    /// Only a client created from an anonymous server's endpoint can communicate with that server. Retrieve the ``NonBlockingServer/endpoint`` and
```

Comment:
This is a bit of a sentence fragment, and doesn't fit in:


> "Retrieve the ``NonBlockingServer/endpoint`` and send it across an existing XPC connection."

Reply:
Yeah it does read a bit awkwardly, I've changed it to:
> Do this by retrieving the server’s `NonBlockingServer/endpoint` and then creating a client with it: